src/app/components/select-field/index.html

```diff
@@ -6,86 +6,93 @@
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="pickerHeading"
+      propertyName="selectMode"
       isOptional="true"
+      defaultValue="multiple"
     >
-    Specifies the modal header. When <stache-code>selectMode</stache-code> is <stache-code>single</stache-code>, the default header is "Select a value." When <stache-code>selectMode</stache-code> is <stache-code>multiple</stache-code>, the default header is "Select values."
+    Specifies the selection mode that determines whether users can select one item or multiple items in the select field component's modal. The valid options are <stache-code>single</stache-code>, which displays a text input and allows users to select one item, and <stache-code>multiple</stache-code>, which displays a button and allows users to select multiple items.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="singleSelectClearButtonTitle"
       isOptional="true"
       defaultValue="Clear selection"
     >
-    Specifies the tooltip text for the icon to clear a selection when the <stache-code>selectMode</stache-code> is <stache-code>single</stache-code>. The clear icon  appears in the select field after users select an item.
+    Specifies tooltip text for the icon that clears the text input when <stache-code>selectMode</stache-code> is set to <stache-code>single</stache-code>. The clear icon appears after users select an item. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="singleSelectOpenButtonTitle"
       isOptional="true"
       defaultValue="Click to select a value"
     >
-    Specifies the tooltip text for the select field when the <stache-code>selectMode</stache-code> is <stache-code>single</stache-code>.
+    Specifies tooltip text for the text input when <stache-code>selectMode</stache-code> is set to <stache-code>single</stache-code>. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="singleSelectPlaceholderText"
       isOptional="true"
       defaultValue="Select a value"
     >
-    Specifies the ghost text to display in the select field appears when the <stache-code>selectMode</stache-code> is <stache-code>single</stache-code> and no item has been selected.
+    Specifies ghost text to display in the text input when <stache-code>selectMode</stache-code> is set to <stache-code>single</stache-code> and no item is selected. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
+  <sky-demo-page-properties sectionHeading="Select field events">
+    <sky-demo-page-property
+      propertyName="blur"
+    >
+      Fires when the select field component's text input or button is touched. This event does not emit a value.
```

Comment:
Can we change "touched" to "selected"? Or would that change the meaning?

Reply:
@Blackbaud-TrevorBurch and @blackbaud-conorwright, now thay https://github.com/blackbaud/skyux-select-field/pull/11/files is merged, can we update the docs? Or is this still pending?

Reply:
Touched/blurred have weird meaning. How about `when the select field component loses focus`? since that is the actual trigger now

Reply:
Works for me. Anything else need to be documented after all the changes?

Reply:
I don't believe so. Should be good to go once the skyux2 release is out